ibis/bigquery/tests/conftest.py

```diff
@@ -0,0 +1,60 @@
+import six
+import pytest
+
+import pandas as pd
+import ibis.util
+import ibis.bigquery
+from ibis.bigquery.client import infer_schema_from_df
+
+
+pytest.importorskip('google.cloud.bigquery')
+pytestmark = pytest.mark.bigquery
+
+
+PROJECT_ID = 'ibis-gbq'
+DATASET_ID = 'ci_{}'.format(ibis.util.guid())
+TABLE_ID = 'table_{}'.format(ibis.util.guid())
+
+
+@pytest.fixture(scope='session')
+def df():
+    return pd.DataFrame(dict(
+        int_column=[1], float_column=[2.],
+        string_column=['a'], bool_column=[True],
+    ))
+
+
+@pytest.fixture(scope='session')
+def client():
+    return ibis.bigquery.connect(PROJECT_ID, DATASET_ID)
+
+
+@pytest.fixture(scope='session')
+def client_with_table(client, df):
+    json_file = six.BytesIO(
+        '\n'.join((v.to_json() for (k, v) in df.iterrows())).encode('UTF-8')
+    )
+    schema = infer_schema_from_df(df)
```

Comment:
I uploaded `functional_alltypes` to the `ibis-gbq` project. Run your tests against that so we don't have to upload anything during tests. Then you don't need to import this function, and you'll be able to pass conda build CI.